book/src/Example.md

```diff
@@ -7,97 +7,241 @@ section shows how to deploy and use that driver in Kubernetes.
 
 ## Deployment
 
-The following .yaml file will deploy a `csi-pod` with the HostPath CSI driver and the
-[sidecar containers](CSI-Kubernetes.html#sidecar-containers) which integrate the driver into Kubernetes.
-Use it with a cluster that is [set up for CSI](Setup.html), either by copying it into a local file or
-by referencing the latest version directly:
-```
-$ kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/docs/master/book/src/example/csi-setup.yaml
-storageclass.storage.k8s.io "csi-hostpath-sc" created
-serviceaccount "csi-service-account" created
-clusterrole.rbac.authorization.k8s.io "csi-cluster-role" created
-clusterrolebinding.rbac.authorization.k8s.io "csi-role-binding" created
-pod "csi-pod" created
-$ kubectl get pods
-NAME      READY     STATUS    RESTARTS   AGE
-csi-pod   4/4       Running   0          24s
+This is tested with Kubernetes v1.12. Set the following feature gate flags to true:
+
+```
+--feature-dates=CSIPersistentVolume=true,MountPropagation=true,VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true,CSINodeInfo=true,CSIDriverRegistry=true
+```
+
+`CSIPersistentVolume` is enabled by default in v1.10. `MountPropagation` is enabled by default
+in v1.10. `VolumeSnapshotDataSource` is a new alpha feature in v1.12. `KubeletPluginsWatcher`
+is enabled by default in v1.12. `CSINodeInfo` and `CSIDriverRegistry` are new alpha features
+in v1.12.
+
+CRD needs to be created manually for `CSIDriverRegistry` and `CSINodeInfo`.
+
+```
+$ kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/v1.12.0/staging/src/k8s.io/csi-api/pkg/crd/testdata/csidriver.yaml --validate=false
```

Comment:
Suggestion: point to the CSI CRD repo instead of staging.  That way updates are automatically sync'd.

https://github.com/kubernetes/csi-api/blob/master/pkg/crd/testdata/csidriver.yaml

Reply:
Done